diff --git a/src/main/java/org/opensearch/neuralsearch/constants/SemanticInfoFieldConstants.java b/src/main/java/org/opensearch/neuralsearch/constants/SemanticInfoFieldConstants.java
index f6bee47..a524d34 100644
--- a/src/main/java/org/opensearch/neuralsearch/constants/SemanticInfoFieldConstants.java
+++ b/src/main/java/org/opensearch/neuralsearch/constants/SemanticInfoFieldConstants.java
@@ -19,7 +19,9 @@ public class SemanticInfoFieldConstants {
     public static final String KNN_VECTOR_DATA_TYPE_FIELD_NAME = "data_type";
     public static final String KNN_VECTOR_METHOD_FIELD_NAME = "method";
     public static final String KNN_VECTOR_METHOD_NAME_FIELD_NAME = "name";
-    public static final String KNN_VECTOR_METHOD_DEFAULT_NAME = "hnsw";
+
+    // Change default method from "hnsw" to "disk_ann" for JVector compatibility
+    public static final String KNN_VECTOR_METHOD_DEFAULT_NAME = "disk_ann";
     public static final String KNN_VECTOR_METHOD_SPACE_TYPE_FIELD_NAME = "space_type";
 
     public static final String CHUNKS_FIELD_NAME = "chunks";
