diff --git a/build.gradle b/build.gradle
index eb97ce0..eb6bf30 100644
--- a/build.gradle
+++ b/build.gradle
@@ -202,46 +202,12 @@ tasks.register("preparePluginPathDirs") {
     }
 }
 
-def pluginName = 'opensearch-neural-search'
-def pluginDescription = 'A plugin that adds dense neural retrieval into the OpenSearch ecosytem'
+def pluginName = 'opensearch-neural-search-jvector'
+def pluginDescription = 'A plugin that adds dense neural retrieval with jvector into the OpenSearch ecosytem'
 def projectPath = 'org.opensearch'
 def pathToPlugin = 'neuralsearch.plugin'
 def pluginClassName = 'NeuralSearch'
 
-publishing {
-    repositories {
-        maven {
-            name = "Snapshots"
-            url = System.getenv("MAVEN_SNAPSHOTS_S3_REPO")
-            credentials(AwsCredentials) {
-                accessKey = System.getenv("AWS_ACCESS_KEY_ID")
-                secretKey = System.getenv("AWS_SECRET_ACCESS_KEY")
-                sessionToken = System.getenv("AWS_SESSION_TOKEN")
-            }
-        }
-    }
-    publications {
-        pluginZip(MavenPublication) { publication ->
-            pom {
-                name = pluginName
-                description = pluginDescription
-                groupId = "org.opensearch.plugin"
-                licenses {
-                    license {
-                        name = "The Apache License, Version 2.0"
-                        url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
-                    }
-                }
-                developers {
-                    developer {
-                        name = "OpenSearch"
-                        url = "https://github.com/opensearch-project/neural-search"
-                    }
-                }
-            }
-        }
-    }
-}
 opensearchplugin {
     name pluginName
     description pluginDescription
diff --git a/settings.gradle b/settings.gradle
index 232fd31..7da24b9 100644
--- a/settings.gradle
+++ b/settings.gradle
@@ -8,7 +8,7 @@
  * This project uses @Incubating APIs which are subject to change.
  */
 
-rootProject.name = 'neural-search'
+rootProject.name = 'neural-search-jvector'
 
 include ":qa"
 include ":qa:rolling-upgrade"
